src/helpers.ts

```diff
@@ -0,0 +1,21 @@
+import { Progress } from './types/Progress';
+import { Todo } from './types/Todo';
+
+export const itemsLeft = (arr: Todo[]) => {
+  return arr.filter(item => item.completed === false).length;
+};
```

Comment:
The name of the function is usually called from the verb
```suggestion
export const getUncompletedTodos = (todos: Todo[]) => {
  return todos.filter(({ completed }) => !completed).length;
};
```

Reply:
not fixed